app/controllers/application_controller.rb

```diff
@@ -8,12 +8,21 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action(
     :authenticate_user!,
-    unless: -> { self.class == HighVoltage::PagesController }
+    unless: lambda do
+      self.class == HighVoltage::PagesController ||
+      self.class == SubscriptionsController
```

Comment:
This should be implemented inside `SubscriptionController` and not here, by overriding the `before_action` there ;) we only put here rules about controllers from external gems.

Reply:
So... I should have a `before_action`inside `SubscriptionsController` to skip authentication and authorization?